src/Item.h

```diff
@@ -101,6 +101,9 @@ class cItem
 	bool IsCustomNameEmpty(void) const { return (m_CustomName.empty()); }
 	bool IsLoreEmpty(void) const { return (m_LoreTable.empty()); }
 
+	/** Get the number of items in the stack. */
+	char GetCount(void) { return m_ItemCount; }
```

Comment:
```m_ItemCount``` is ```public``` for ```cItem```.

So do ```Item.m_ItemCount```
Or add the function to the LuaAPIDoc

Reply:
How do I do that? I'd guess it probably should be exposed to plugins

Reply:
This is accessible by lua - the member is also bublic there

Any LuaAPI aditions have to be added in Server/Plugins/APIDump
either in APIDesc.lua or in one of the folders mathching the class. And they should be kept in alphabetical order

Reply:
I won't make the member private but I think we should encourage the use of getters/setters as a safer way to interact with plugins.

Reply:
Yes You are right. I also wondering about that.

I think the best is to open a thread on the forum and lay down your thoughts there so that everyone can comment on that as items are quite essential to the gameplay/server